broker-node/app/ChunkEvents.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+
+namespace App;
+
+
+
+use App\Clients\BrokerNode;
+
+use Illuminate\Database\Eloquent\Model;
+
+use Webpatser\Uuid\Uuid;
+
+
+
+class ChunkEvents extends Model
+
+{
+
+
+    public static function boot()
+    {
+
+        parent::boot();
+
+        self::creating(function ($model) {
+
+            $model->id = (string)Uuid::generate(4);
+
+        });
+
+    }
+    
+    protected $table = 'chunk_events';
+    protected $fillable = [
+        'hook_node_id',
+        'session_id',
+        'event_name',
+        'value'
+    ];
+    public $incrementing = false;  // UUID
+    
+
+	
+	public static function addChunkEvent($event_name, $hook_node_id, $session_id, $value){
```

Comment:
Make this `private`. 

Also, not sure if this abstraction is needed. It becomes less clear which values are saved since you rely on parameter position rather than an associative array.